### DISCUSSION TITLE
How to customize the theming for light and dark mode

### DISCUSSION THREAD

**Discussion by kapi-caz** [carbon-design-system/carbon#17730] [Help] [answered]
Hi everyone,

I want to have custom light and dark mode. Hence, I overwrite the tokens of the given themes `white`, `g10`, `g90` and `g100`. I found an [issue](https://github.com/carbon-design-system/carbon/issues/13007) which addresses this, but I can't make it work in my project. 
I've created a [stackblitz](https://stackblitz.com/edit/github-bcbtt7?file=light-dark-mode%2Fsrc%2Fscss%2Fstyles.scss) with the carbon example `light-dark-mode`. 
My simple themes for light and dark do just define the `background` and the `text-primary` colour. This works partly, but not inside the `<section className=theme-section>`. 

When I inspect it, I can see that the `<div class="cds--g10 cds--layer-one">` overwrites the two tokens:
In the HTML tag:
<img width="896" alt="image" src="https://github.com/user-attachments/assets/f38ee618-4e1b-4849-a037-ae479ec26b38">

Inside the div node `cds--g10`:
<img width="983" alt="image" src="https://github.com/user-attachments/assets/057c54b4-3244-4dbd-a2e7-fe804803f5c8">

> Note: the div with the class `cds--toggle` works as expected and the toggle button text color changes as well as the background around the nested box `theme-section`.

Any hints or ideas are welcome :) . I would love to understand my misconfiguration here in this example to debug my own project.

Thanks and best,

Kapilan

**Comment by janhassel** [comment, marked as answer]
Hey, you should be able to customize the themes by configuring them with Sass:

```scss
@use '@carbon/react/scss/themes' with (
  $g10: (
    background: #ff0000,
  ),
);
```

**Reply by kapi-caz** [reply]
Thank you @janhassel!

This works! I've updated my stackblitz accordingly. 

Best,
Kapi
